### ISSUE TITLE
Add ARIA roles and attributes to the range slider component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#160]
### Problem

[molecules/molecules.range-slider](https://a-ui.github.io/core_branding_scss/#molecules-range-slider)

The markup of the range slider is missing ARIA in order to work properly with assitive technologies such as screenreaders.

The same applies to the multi-thumb slider.

### Solution

Follow the ARIA design pattern for slider and multi-thumb slider.

### Articles and links

* [ARIA design pattern: Slider](https://www.w3.org/TR/wai-aria-practices-1.2/#slider)
* [ARIA design pattern: Slider (Multi-Thumb)](https://www.w3.org/TR/wai-aria-practices-1.2/#slidertwothumb)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
